dm/metrics/grafana/DM-Monitor-Professional.json

```diff
@@ -1,45 +1,9 @@
 {
-  "__inputs": [
-    {
-      "name": "DS_TEST-CLUSTER",
-      "label": "prometheus-local",
-      "description": "",
-      "type": "datasource",
-      "pluginId": "prometheus",
-      "pluginName": "Prometheus"
-    }
-  ],
-  "__requires": [
-    {
-      "type": "grafana",
-      "id": "grafana",
-      "name": "Grafana",
-      "version": "7.5.7"
-    },
-    {
-      "type": "panel",
-      "id": "graph",
-      "name": "Graph",
-      "version": ""
-    },
-    {
-      "type": "datasource",
-      "id": "prometheus",
-      "name": "Prometheus",
-      "version": "1.0.0"
-    },
-    {
-      "type": "panel",
-      "id": "timeseries",
-      "name": "Time series",
-      "version": ""
-    }
-  ],
   "annotations": {
     "list": [
       {
         "builtIn": 1,
-        "datasource": "${DS_TEST-CLUSTER}",
```

Comment:
not sure whether we should change this placeholder which may used on cloud or tiup monitoring which has cluster name.

also this file change is so large that it's very hard to review what's changed. @lance6716 how is this file changed in previous pr?

Reply:
I changed it by mistake and I'm looking for a solution now.

Reply:
ehco's work, this file is related to many CI CD, should talk with them first.

Reply:
I've asked @Ehco1996 for help. This file can be exported from the grafana server, where, however, the `datasource` should be `${DS_TEST-CLUSTER}`, a placeholder, rather than a specific cluster name.

I'll fix it in the next commit.

Reply:
can this file be smaller? hard to review

Reply:
@Ehco1996